compositor_pipeline/src/pipeline/decoder/audio.rs

```diff
@@ -57,7 +66,8 @@ pub fn start_audio_decoder_thread(
                 Level::INFO,
                 "Audio decoder",
                 input_id = input_id.to_string()
-            );
+            )
+            .entered();
```

Comment:
This was not working before, span has to be entered to have effect https://docs.rs/tracing/latest/tracing/span/index.html